### PR TITLE
Added AP Reports model

### PR DESCRIPTION
**Pull request by sumitsourav**
**Added AP Reports model to the gem:**

1. AP aging header
2. daily payable outstanding
3. payable coming due
4. payable summary

### CODE REVIEW

File: app/platform_api/schema/ap_aging_header_info.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  # @type: number
+  # @format: double
+  field :total_ap_amount
+
+  # Portion of Total AR this data represents.
+  # @type: number
+  # @format: double
+  field :percentage_of_total_ap
+
+
```
Comment (lockstep-rakesh) [inline app/platform_api/schema/ap_aging_header_info.rb:60]: can you pls remove this extra lines

File: app/platform_api/schema/daily_payable_outstanding_report.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  # @type: integer
+  # @format: int32
+  field :invoice_count
+
+  # Time (in days) between an invoice was completed paid off and when the invoice was issued
+  # @type: number
+  # @format: double
+  field :daily_payable_sales_outstanding
+
+
```
Comment (lockstep-rakesh) [inline app/platform_api/schema/daily_payable_outstanding_report.rb:23]: same here and check for all schema files

File: app/models/lockstep/report_ap_aging_header.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Lockstep::ReportApAgingHeader < Lockstep::ApiRecord
+  self.model_name_uri = "v1/Reports/ap-aging-header"
+  self.id_ref = "group_key"
+  self.query_path=""
```
Comment (lockstep-rakesh) [inline app/models/lockstep/report_ap_aging_header.rb:4]: can you pls add space - self.query_path = "", check this across

File: app/platform_api/schema/ap_aging_header_info.rb (hunk 1 above)
Comment (sumitsourav) [reply app/platform_api/schema/ap_aging_header_info.rb]: Resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/3/commits/bb7622d4d99eb2976f7875442d449eed55f1fcae)

File: app/models/lockstep/report_ap_aging_header.rb (hunk 3 above)
Comment (sumitsourav) [reply app/models/lockstep/report_ap_aging_header.rb]: Resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/3/commits/bb7622d4d99eb2976f7875442d449eed55f1fcae)

**Pull request merged**